jobs/log-cache-cf-auth-proxy/templates/bpm.yml.erb

```diff
@@ -3,7 +3,14 @@
   certDir = "#{jobDir}/config/certs"
 
   gw = link('log-cache-gateway')
-  cc = link('cloud_controller')
+  cc_address = ""
+  if_p("cc.address") { |addr|
+    cc_address = addr
+  }.else_if_link('cloud_controller') { |cc|
+    cc = link('cloud_controller')
+    cc_address = cc.addr
```

Comment:
This is still wrong: Re: from erros from last integration run.

Reply:
can you show me the latest errors?  Because this now exactly matches the code that was in this PR in loggregator-release (https://github.com/cloudfoundry/loggregator-release/pull/413/files) where this worked.

So I'm confused why this isn't working in this tests.

Reply:
![image](https://user-images.githubusercontent.com/2761008/117490264-cf38e900-af2b-11eb-98d2-7061723872f7.png)
it actually doesn't match. As mentioned in the error posted above: 
![image](https://user-images.githubusercontent.com/2761008/117490332-df50c880-af2b-11eb-90fc-42d2457412ef.png)
add -> address

Reply:
(in so far as tests, log-cache doesn't have template tests)

Reply:
I need a little more context you pasted in an image which I don't know where you got that image showing "cc.address". there is 2 things.  cc.address - which is if you override and "cc.addr" which is VALID if you have a bosh link to the cloud_controller.  

I do not know what "line 12" is as this is not a valid line in the bpm.yml.erb as line 12 is just a close '}'

I'm looking at the loggregator-release/jobs/reverse_log_proxy_gateway/template/bpm.yml.erb and I see it is the same as what I have put in this PR.